getting_serious/puzzle_29.R

```diff
@@ -0,0 +1,47 @@
+## Ten people, two rooms, states are {0, 1, 2, ..., 10} (number of people in other room)
+states <- seq(0, 10)
```

Comment:
What about defining `n <- 10`? That would allow for a few abbreviations in the rest of the code. But then there should also be an abbreviation for `length(states)`, and that could push the number of local variables beyond the short-term memory limit.

Reply:
Good idea, I like the idea of having a `total_n_people` or something like that